libs/SFMLResource/GraphicsTextureResource.hpp

```diff
@@ -0,0 +1,31 @@
+/*
+** EPITECH PROJECT, 2022
+** rtype
+** File description:
+** GraphicsTextureResource
+*/
+
+#ifndef GRAPHICSTEXTURERESOURCE_HPP_
+#define GRAPHICSTEXTURERESOURCE_HPP_
+
+#include "Resource/Resource.hpp"
+#include <SFML/Graphics.hpp>
+
+namespace ecs
+{
+    /// @brief This resource class stores a Graphical SFML texture to be set on a rectangle shape resource.
+    /// This class is created in order to set a texture on a rectangle shape resource.
+    class GraphicsTextureResource : public Resource {
+        public:
+            /// @brief The graphical SFML texture to be set on a rectangle shape resource.
+            sf::Texture texture;
```

Comment:
Can you not specify the targeted file (.png or other picture extension) to its creation? 
If possible, you should add the possibility to do so.

Reply:
I add a constructor with the possibility to pass a file as parameters